src/scss/responsive/_layout.scss

```diff
@@ -0,0 +1,8 @@
+.container {
+  @media only screen and (min-width: $tabletMin) and (max-width: $tabletMax) {
+    width: calc(100% - 60px);
+  }
+  @media only screen and (max-width: $mobileMax) {
+    width: calc(100% - 40px);
```

Comment:
The value o `60px` or `40px` is like a magic number, this can be better by define a constant for it, something like:
- $space-large
- $space-default
- $space-small